app/scripts/views/header.coffee

```diff
@@ -117,6 +122,19 @@ module.exports = class HeaderView extends View
     return if utils.isLogged()
     @subview("dialog-login").show()
 
+  anonymousMode: (e) ->
+    e.preventDefault()
+    ViewHelpers.toggleAnonymousStatus()
+
+  RefreshAnonymousStatus: ->
+    anonymous = ViewHelpers.getAnonymousModeStatus()
+    icon = @$(".icon-eye-close")
```

Comment:
Better to add some meaningful class like `anonymous-mode-icon`.
Also, note you will fail here if user is not logged (see comment above).